src/app/heroes/heroes.component.scss

```diff
@@ -0,0 +1,49 @@
+/* HeroesComponent's private CSS styles */
+:root {
+  --height:15px;
+}
+
+.heroes {
```

Comment:
The nice thing about `scss` is that you can nest the `css` to make it cleaner and take up fewer lines. This file should look more like this:
```
.heroes {
    // heroes styles...
    li {
        // li styles...
        &:hover {
            // li:hover styles here...
        }
        // ...       
    }
}
```
I know this was probably just taken directly from the heroes tutorial. Can you rewrite this file in the flavor of `scss`?